content/reference/template-functions/static-context.md

```diff
@@ -159,6 +158,13 @@ Or for a total of 64 `a`s and `b`s:
 '{{repl RandomString 64 "[ab]" }}'
 ```
 
+Each time that this function is called, the behavior changes based on the [hidden](/reference/v1beta1/config/#hidden) and [readonly](/reference/v1beta1/config/#readonly) properties.
```

Comment:
does hidden matter? i dont know

Reply:
It does not but generally it is used with `hidden`.